packages/pwa/app/components/cart-item-variant/item-price.jsx

```diff
@@ -39,7 +40,7 @@ const ItemPrice = ({currency, ...props}) => {
             <Text fontWeight="bold">
                 <FormattedNumber
                     style="currency"
-                    currency={currency || basket.currency || DEFAULT_CURRENCY}
+                    currency={currency || basket.currency || getCurrency(DEFAULT_LOCALE)}
```

Comment:
You can probably dry this up, assign a variable for this derived currency. Same goes for line 57 and 68